app/src/main/java/com/teamwss/websoso/ui/main/myPage/myLibrary/MyLibraryFragment.kt

```diff
@@ -33,6 +34,7 @@ class MyLibraryFragment : BaseFragment<FragmentMyLibraryBinding>(R.layout.fragme
 
         setupRestGenrePreferenceAdapter()
         setUpObserve()
+        navigateToStorageActivity()
```

Comment:
r: 
```suggestion
        onStroageButtonClick
```
요런식의 네이밍으로 지어야 할 듯요!